### ISSUE TITLE
Optionally output as json

### ISSUE THREAD

**Issue by wdecoster** [wdecoster/cramino#5]
Optionally output the metrics as a (pretty printed) json

**Comment by mbhall88** [comment]
Indeed, a machine-readable output format would be fantastic. Even something like a CSV would be nice.

**Comment by wdecoster** [comment]
The latest release (v1.0.0, in a few minutes) adds a --format option, which is text (by default), tsv, or json
